index.html

```diff
@@ -1494,6 +1494,59 @@ <h3>Modifications to existing procedures</h3>
       </p>
     </section>
   </section>
+  <section id="always-negotiating-datachannels">
+    <h2>Always negotiating data channels</h2>
+    <section id="always-negotiating-datachannels-configuration">
+      <h3>
+        {{RTCConfiguration}} extensions
+      </h3>
+      <p>
+        {{RTCConfiguration/alwaysNegotiateDataChannels}} defines a way for the application
+        to negotiate datachannels in the SDP offer before creating a datachannel.
+      </p>
+      <pre class="idl">partial dictionary RTCConfiguration {
+    boolean alwaysNegotiateDataChannels = false;
+};</pre>
+      <section>
+        <h2>
+          Dictionary {{RTCConfiguration}} Members
+        </h2>
+        <dl data-link-for="RTCConfiguration" data-dfn-for="RTCConfiguration" class="dictionary-members">
+          <dt>
+            <dfn data-idl="">alwaysNegotiateDataChannels</dfn> of type <span class="idlMemberType">boolean</span>
+          </dt>
+          <dd>
+            <p>
+            Specifies whether the application prefers to always negotiate data channels in the initial SDP offer.
+            </p>
+          </dd>
+        </dl>
+      </section>
+    </section>
+    <section id="always-negotiating-datachannels-interface-extensions">
+      <h3>
+        JSEP modifications
+      </h3>
+      <p>
+        In the PeerConnection's Constructor in [[RTCWEB-JSEP]] section 4.1.1, add
+        "The application can specify whether it prefers to always negotiate datachannels"</li>
+      </p>
+      <p>
+        In the algorithms for generating initial offers in [[RTCWEB-JSEP]] section 5.2.1, insert
+        "If the application prefers to always negotiate datachannels, a m=section MUST be generated for data with fields set as described below."
```

Comment:
A JSEP revision would move the description from below but for a modification this is easier.